src/contracts/utils.ts

```diff
@@ -0,0 +1,53 @@
+import assert from "assert";
+import { BigNumber, BigNumberish, Event } from "ethers";
+import { isDefined } from "../utils";
+
+/**
+ * Migrated directly from @uma/sdk.
```

Comment:
Should we document the function and leave this as a `@note`?

Reply:
Updated here: https://github.com/across-protocol/sdk-v2/pull/389/commits/51f2336d2fe1fb987d4a0de4a548904716af95dd